### ISSUE TITLE
Migrate from migration *.sql files not working if only *.sql files

### ISSUE THREAD

**Issue by Daavidaviid** [go-pg/migrations#42]
Hi,
I don't know if I'm wrong but it looks like you can't have all migrations in `.sql` files only, you have to have at least one migration in a `.go` file.

You can reproduce this by cloning the example folder and removing all but the `*.sql` files.

A workaround I found is to create a dumb `0_dumb.up.go`file doing nothing as you can see : 
```
package main

import (
  "errors"
  "fmt"

  "github.com/go-pg/migrations"
)

func init() {
  migrations.MustRegisterTx(func(db migrations.DB) error {
    fmt.Println("Dumb")
    return errors.New("Dumb")
  }, func(db migrations.DB) error {
    fmt.Println("Dumb")
    return errors.New("Dumb")
  })
}

```

And only then it works.
I'm using the latest version as of today and my go version is `go1.11 darwin/amd64`

**Comment by vmihailenco** [comment]
Should be fixed in v6.4.2

**Comment by Daavidaviid** [comment]
Thanks !
